src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.verified.txt

```diff
@@ -1486,6 +1510,16 @@ namespace Akka.Actor
             public static readonly Akka.Actor.ProviderSelection.Remote Instance;
         }
     }
+    public sealed class Reason
+    {
+        public Reason(string type, string message = null) { }
+        public string Message { get; }
+        public string Type { get; }
+    }
+    public class static ReasonExtensions
+    {
+        public static Akka.Actor.Reason ToReason(this System.Exception ex) { }
+    }
```

Comment:
This extensions method should be on the ActorContext or ActorSystem,
to make it possible to retrieve a new AkkaErrorService from Extensions list.
The Default implementation can simply convert the `Exception` to a `Reason` with `Error = Guid.Empty`

Reply:
These metrics are only produced within the 'ActorCell' and it's performed automatically by Akka.NET